### PR TITLE
[dev-v2.7] rancher-backup add 102.0.4+up3.1.4-rc1 

### PR DESCRIPTION
**Pull request by mallardduck**

### CODE REVIEW

**Comment by joshmeranda** [comment]
@mallardduck is the upstream version also an rc? If so do we need a chart for it or can it wait until the upstream is un-rcd?

File: packages/rancher-backup/rancher-backup-crd/package.yaml [hunk 1]
```diff
@@ -1,2 +1,2 @@
-url: https://github.com/rancher/backup-restore-operator/releases/download/v3.1.3/rancher-backup-crd-3.1.3.tgz
-version: 102.0.3
+url: https://github.com/rancher/backup-restore-operator/releases/download/v3.1.4-rc1/rancher-backup-crd-3.1.4-rc1.tgz
+version: 102.0.4-rc1
```
Comment (alexandreLamarre) [inline packages/rancher-backup/rancher-backup-crd/package.yaml]: Looks like per previous PRs, this should `102.0.4`

File: packages/rancher-backup/rancher-backup/package.yaml [hunk 2]
```diff
@@ -1,2 +1,2 @@
-url: https://github.com/rancher/backup-restore-operator/releases/download/v3.1.3/rancher-backup-3.1.3.tgz
-version: 102.0.3
+url: https://github.com/rancher/backup-restore-operator/releases/download/v3.1.4-rc1/rancher-backup-3.1.4-rc1.tgz
+version: 102.0.4-rc1
```
Comment (alexandreLamarre) [inline packages/rancher-backup/rancher-backup/package.yaml]: Same here

File: release.yaml [hunk 3]
```diff
@@ -1,3 +1,13 @@
+fleet:
+  - 102.2.4+up0.8.4-rc.1
+fleet-agent:
+  - 102.2.4+up0.8.4-rc.1
+fleet-crd:
+  - 102.2.4+up0.8.4-rc.1
+rancher-backup:
+  - 102.0.4-rc1+up3.1.4-rc1
+rancher-backup-crd:
+  - 102.0.4-rc1+up3.1.4-rc1
```
Comment (alexandreLamarre) [inline release.yaml]: should be `102.0.4+up3.1.4-rc1`

**Review by alexandreLamarre — CHANGES_REQUESTED** [review]

**Comment by mallardduck** [comment]
@joshmeranda - there we go that looks more like past RC releases for BRO now.

**Review by alexandreLamarre — APPROVED** [review]

**Review by lucasmlp — APPROVED** [review]
LGTM 🚀

**Pull request merged**